Makefile

```diff
@@ -0,0 +1,22 @@
+# SPDX-License-Identifier: Apache-2.0
+# Copyright 2022-present Open Networking Foundation
+
+PROJECT_NAME             := pfcpsim
+VERSION                  ?= $(shell cat ./VERSION)
+
+## Docker related
+DOCKER_REGISTRY          ?=
+DOCKER_REPOSITORY        ?=
+DOCKER_TAG               ?= ${VERSION}
+DOCKER_IMAGENAME         := ${DOCKER_REGISTRY}${DOCKER_REPOSITORY}${PROJECT_NAME}:${DOCKER_TAG}
+DOCKER_BUILDKIT          ?= 1
+
+DOCKER_TARGET           ?= pfcpsim-client
+
+run-pfcpsim-client:
```

Comment:
do we really need this? it looks like it's used for testing purposes only

Reply:
ops, leftover